frontend/src/e2e-test/pages/employee/person-search.ts

```diff
@@ -66,6 +67,7 @@ export default class PersonSearchPage {
     await this.#createPersonModal.postalCodeInput.type(personData.postalCode)
     await this.#createPersonModal.postOfficeInput.type(personData.postOffice)
     await this.#modalConfirm.click()
+    await waitUntilFalse(() => this.#createPersonModal.modal.visible)
```

Comment:
Suggestion: `this.#createPersonModal.waitUntilHidden()`

Also, I think it would be good to pair a "negative test" with a corresponding positive test (e.g. `waitUntilVisible()` before the click), to reduce the possibility of some later refactoring making the negative test pass always without actually checking anything.